### PR TITLE
Js3 week2 homework

### PR DESCRIPTION
**Pull request by Ann-Grabetski**
I could not get this code to work. I do not understand how to do this.

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -29,26 +33,149 @@
...
+      const repo = repos[select.value];
+      renderRepoDetails(repo, mainSection);
+
+      const contributorsSection = createAndAppend('section', mainSection, {
+        class: 'contributors-container',
+      });
+      createAndAppend('h3', contributorsSection, { text: 'Contributions' });
+      const contributorsList = createAndAppend('ul', contributorsSection);
+
+      const CONTRIBUTORS_URL = `${HYF_BASE_URL}/repos/HackYourFuture/${repo.name}/contributors`;
```
Comment (remarcmij) [inline homework/index.js]: You should not construct this url yourself. Use the `contributors_url` from the repo object.

File: homework/index.js [hunk 2]
```diff
@@ -29,26 +33,149 @@
...
-      repos.forEach(repo => renderRepoDetails(repo, ul));
-    });
+        }),
+      );
   }
 
-  const HYF_REPOS_URL =
-    'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
-  window.onload = () => main(HYF_REPOS_URL);
+  //   const mainSection = createAndAppend(`main`, root, {
```
Comment (remarcmij) [inline homework/index.js]: Dead code like this should never be left in a PR. Nobody is interested in what you tried and abandoned. And it certainly should not makes its way into production code, if this PR were to be merged into the master branch.

Comment (Ann-Grabetski) [reply homework/index.js]: Oh, sorry. I just forgot about this piece.

File: homework/index.js [hunk 3]
```diff
@@ -29,26 +33,149 @@
...
   function main(url) {
-    fetchJSON(url, (err, repos) => {
-      const root = document.getElementById('root');
-      if (err) {
+    const root = document.getElementById('root');
+
+    const header = createAndAppend(`header`, root);
+    createAndAppend(`h1`, header, { text: `HYF Repositories` });
+    const select = createAndAppend(`select`, header);
+    createAndAppend('option', select, {
```
[the remark this answers is not in the document]
Comment (Ann-Grabetski) [reply homework/index.js]: Oops.. I'll fix it

File: homework/index.js (hunk 1 above)
Comment (Ann-Grabetski) [reply homework/index.js]: I'll try

**Pull request closed without merging**